### DISCUSSION TITLE
Docs Improvement

### DISCUSSION THREAD

**Discussion by EnokiUN** [EnokiUN/voltage#44] [General]
Here we will discuss working on and improving the docs I suppose.

**Comment by EnokiUN** [comment]
@ksIsCute

**Comment by siqnole** [comment]
wooo discussion fuck yeah

**Comment by siqnole** [comment]
ok so what are our options so far
(e.g rtd.io, pages, site, other stuff?)

**Reply by EnokiUN** [reply]
readthedocs, pages didn't work that well yesterday.

It's just readthedocs being weird with docutils which are used for attributetable.

**Reply by siqnole** [reply]
docutils rolled ig

**Comment by EnokiUN** [comment]
WOOO, DOCS WORK NOW!!!!

ALL I HAD TO DO WAS ADD ONE LIKE TO `.readthedocs.yml`

I'm not happy :(

**Reply by siqnole** [reply]
still works

**Comment by siqnole** [comment]
LETS GOOOOOOOOOOOOoo
